css/showcase.css

```diff
@@ -0,0 +1,77 @@
+.showcase-body {
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+	max-width: 80ch;
+	width: 100%;
+	transition: all 0.5s ease;
+}
+
+.showcase-list-item {
+	margin: 0.5rem 0 0.5rem 2rem;
+	cursor: pointer;
+	transition: all 0.5s ease;
+}
```

Comment:
Hey zoef similar to this there are multiple places where you have used rem for anything below 10px. I can see this for margin, padding and font-size. I would suggest to use px for this. rest seems good for me.